### PR TITLE
Upasana new

### PR DESCRIPTION
**Pull request by banerjeeupasana03**
Please help me fix the animation issues.

### CODE REVIEW

File: comps/Button.js [hunk 1]
```diff
@@ -7,20 +7,25 @@ import {
...
   Button,
-  Alert
+  Alert,
+  Animated
 } from 'react-native';
 // import Styles from '../styles/BtnStyle';
 
 function Btn(){
+
+    const[btnheight, SetBtnheight] = useState[20];
```
Comment (AndriusCvetkovas) [inline comps/Button.js]: const [btnHeight] = useState(new Animated.Value(1));

File: comps/Button.js [hunk 2]
```diff
@@ -7,20 +7,25 @@ import {
...
   return(
-    <View>
+    <Animated.View>
       <Button
           color='green'
           
           title="Animated"
-          onPress={() => Alert.alert('Simple Button pressed')}
+          onPress={() => Alert.alert('Simple Button pressed')
+            SetBtnheight = 40;
```
Comment (AndriusCvetkovas) [inline comps/Button.js]: Animated.timing(
            btnHeight,
            {
                toValue:2,
                duration:500
            }
        ).start()

File: comps/Button.js [hunk 3]
```diff
@@ -14,18 +14,27 @@ import {
...
+        }
+        ).start()
+
   return(
-    <Animated.View>
+      
+    <View>
       <Button
-          color='green'
+          color='#84B2C3'
```
Comment (AndriusCvetkovas) [inline comps/Button.js]: the height is not specified, height: btnHeight

**Comment by banerjeeupasana03** [comment]
Help needed in fixing my page

File: comps/Button.js [hunk 4]
```diff
@@ -26,15 +27,17 @@ function Btn(){
...
       
-    <View>
-      <Button
+    <Animated.View style = {{width: '30%'}}>
+      {/* <Button */}
+      <TouchableOpacity>
           color='#84B2C3'
-          
           title="Animated"
+          height={btnheight}
```
Comment (AndriusCvetkovas) [inline comps/Button.js]: because u r using touchable opacity color and hight has to be inside the

**Comment by tashavithaya** [comment]
<View style={{flex:1}}>

    <TouchableOpacity
      style={{flexDirection:"row", backgroundColor:"#4AF", width:"50%", height:"8%", borderRadius:10, padding:10, alignSelf:"center"}}>
            <Image source={{
            uri:"https://static.thenounproject.com/png/68669-200.png"}}
            style={{width:25, height:25}}/>
          <Text style={{color:"white", padding:5, fontSize:16}}>send</Text>
    </TouchableOpacity>
    </View>

**Pull request merged**